ee/hogai/graph/root/nodes.py

```diff
@@ -544,6 +562,8 @@ def router(self, state: AssistantState) -> RouteName:
         last_message = state.messages[-1]
 
         if isinstance(last_message, AssistantToolCallMessage):
+            if last_message.ui_payload and "create_support_ticket" in last_message.ui_payload:
+                return "end"
```

Comment:
this is to stop another ai message after returning the tool output. The tool already returns a sensible message and anything extra just adds noise/confusion as it often suggests it's raised a support ticket, when actually the user has to do that